src/App.jsx

```diff
@@ -14,57 +16,36 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectProduct, setSelectProduct] = useState('Jam');
```

Comment:
```suggestion
  const [selectedProduct, setSelectedProduct] = useState('Jam');
```
select is a verb
it is better to use nouns for the state